src/stactools/sentinel1/grd/product_metadata.py

```diff
@@ -2,7 +2,8 @@
 from typing import Any, Callable, Dict, List, Optional
 
 from pystac.utils import str_to_datetime
-from shapely.geometry import Polygon, mapping  # type: ignore
+from shapely.geometry import MultiPolygon  # type: ignore
+from shapely.geometry import Polygon, mapping
```

Comment:
These imports can be consolidated onto one line.
```suggestion
from shapely.geometry import Polygon, MultiPolygon, mapping  # type: ignore
```